### DISCUSSION TITLE
A suggestion for UI

### DISCUSSION THREAD

**Discussion by teamoTNT** [apache/skywalking#10708] [General]
When I upgraded skywaling to the latest version(9.4.0), I found that I couldn't do a global search based on traceId. It selects the first service by default, which was not the case in older versions.
So will future versions consider adding the global search function back?

![image](https://user-images.githubusercontent.com/103160892/233558546-209d2305-b19b-4b91-bfe0-8bb59e85e1b6.png)

**Comment by wu-sheng** [comment]
No, we are not planning to add it back. The size of the trace dataset from our users could be easily about 10b+ in the TTL period. It is costing too much to open this.

**Reply by teamoTNT** [reply]
ok, I understand it. think.
